ethpm/assets/release-package.schema.v2.json

```diff
@@ -0,0 +1,279 @@
+{
+  "title": "Release Package Specification",
+  "type": "object",
+  "required": [
+    "package_version",
+    "package_name",
+    "version"
+  ],
+  "version": "2",
+  "additionalProperties": false,
+  "properties": {
+    "package_version": {
+      "type": "string",
+      "title": "Package Version",
+      "default": "2",
+      "enum": ["2"]
```

Comment:
My idea was that this will eventually be the schema for V2. But I'm worried that "package_version" might get confusing with the "version" field below? Is there a better name we can use?

Reply:
Spec Version?

Also we can drop the `release-` prefix on this file since it's not just a "package"

Reply:
Sounds good to me!